contents/docs/deploy.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Deploy PostHog
+sidebar: Docs
+---
+
+PostHog offers a number of different deployment options:
+
+-   [PostHog Cloud](#posthog-cloud) - Create a new PostHog Cloud deployment with just a few clicks.
+    -   Recommended as it's **the fastest way** to try PostHog out.
+    -   Comes with **extra features for reliability and security** and automatic updates.
+-   [Hobby](#hobby-deployment) - Recommended for testing or very small hobby projects _(&lt;100k events/month)_
+-   [PostHog Self-hosted](#posthog-self-hosted) - Recommended for most self-host deployments.
+
+## PostHog Cloud
+
+[PostHog Cloud](/pricing) is our managed hosting service and let's you start using PostHog in only a few clicks.
+We generally recommend this option for most users as it's the easiest to try out and offers the most features out of the box:
+
+-   **1 million events included _free_ every month!**
+-   Highly-scalable infrastructure to handle large event volumes _(>100m events/month)_
+-   Automatic updates when new features are released
+-   Unlimited teammates and SSO/SAML _(Enterprise only)_
+
+<div class="flex flex-col items-stretch lg:flex-row lg:items-center justify-between bg-gray-accent-light py-4 px-6 rounded-sm mb-4 space-y-3 lg:space-y-0 lg:space-x-3">
+    <div class="space-y-1">
+        <span class="font-semibold text-lg block">Get started with PostHog Cloud.</span>
+        <span class="text-base text-gray block">The easiest and quickest way to set-up PostHog.</span>
+    </div>
+    <CallToAction class="shrink-0" to="/signup">Deploy now</CallToAction>
+</div>
+
+For more information, check out our [pricing](/pricing) page.
+
+## Hobby deployment
+
+The [Hobby](/docs/self-host/hobby) deployment is the most basic method for hosting PostHog yourself, and is suitable for testing or very small projects.
+This deployment runs the entire PostHog stack on a single VM, meaning it can only handle relatively low event volumes (&lt;100k events/month).
+
+For more information on setting up a hobby deployment, check out [here](/docs/self-host/hobby) for instructions.
+
+## PostHog Self-hosted
+
+[PostHog Self-hosted](/docs/self-host) is the recommended method for hosting PostHog yourself and gives much better performance than the Hobby deployment.
+
+This method offers a number of benefits over PostHog Cloud:
+
+-   Customer data never leaves your infrastructure
+-   First party cookies on your own domain
+-   Full SQL access to your data _(Enterprise only)_
+
+However, this also means that you will need to manage all updates, scaling, and security by yourself.
```

Comment:
```suggestion
However, this also means that you will need to manage all updates, scaling, and security by yourself, or engage a partner from [the PostHog Marketplace](/marketplace) to assist you. 
```

Feel free to remove this if you disagree, just feels worth adding another option.